terraform/shared/services/ecs-cluster/main.tf

```diff
@@ -27,6 +26,11 @@ resource "aws_ecs_cluster" "cluster" {
   }
 }
 
+resource "aws_ecs_cluster_capacity_providers" "cluster" {
+  cluster_name       = aws_ecs_cluster.cluster.name
+  capacity_providers = ["FARGATE", "FARGATE_SPOT"]
+}
```

Comment:
I had to make this change because otherwise there was an error.

docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_cluster_capacity_providers